makefile

```diff
@@ -1,2 +1,32 @@
+help:
+	@echo "Please use \`make <target>\` where <target> is one of"
+	@echo "  help                       display this help message"
+	@echo "  migrate                    apply database migrations"
+	@echo "  prod_requirements          install production requirements for edly_panel python3 environment"
+	@echo "  quality                    check code quality for edly_panel"
+	@echo "  requirements               install local requirements for edly_panel python3 environment"
+	@echo "  serve                      serve edly_panel App at 0.0.0.0:9999"
+	@echo "  static                     build and compress static assets"
+	@echo "  test                       run tests"
+	@echo ""
+
 quality:
 	isort -rc --atomic .
+
+requirements:
+	pip install -qr requirements/local.txt --exists-action w
+
+prod_requirements:
+	pip install -qr requirements/production.txt --exists-action w
+
+static:
+	python manage.py collectstatic --noinput
+
+serve:
+	python manage.py runserver 127.0.0.1:8000
```

Comment:
`127.0.0.1:8000` -> `0.0.0.0:8000` (this way you can access you local site from any computer from same network.)

Reply:
Done.